docs/configuration/config-file-reference.md

```diff
@@ -2553,6 +2553,11 @@ The `limits_config` configures default and per-tenant limits imposed by Cortex s
 # CLI flag: -validation.max-label-names-per-series
 [max_label_names_per_series: <int> | default = 30]
 
+# Maximum combined size in bytes of all labels and label values accepted for a
+# series. 0 to disable the limit.
+# CLI flag: -validation.max-series-label-size-bytes
+[max_series_label_size_bytes: <int> | default = 0]
```

Comment:
I think we should be in consistent naming with other label limits and have the config name to be `max_labels_size_bytes`

Reply:
Renamed the flat to `max_labels_size_bytes`